.gitignore

```
@@ -1,25 +1,8 @@
-node_modules
-/dist
-.storybook/dist
-playground/*.map
-
+# Editors
 .idea
```

This should maybe stay here

```suggestion
.idea
.vscode
```